config/default.js

```diff
@@ -30,6 +30,9 @@ module.exports = {
     API_ENDPOINT: '/api',
     'login-redirect': '/',
   },
+  login: {
+    url: '', // TODO point this to Journal
```

Comment:
Yep @diversemix, this PR only gets things working in development/testing

Before things will work in demo/production, this line will need to be change as part of #637